### DISCUSSION TITLE
Help with changing timezone

### DISCUSSION THREAD

**Discussion by rossegit** [DefectDojo/django-DefectDojo#9214] [General]
I am trying to change the time zone from UTC to CST in my docker instance of Defect Dojo.
I have changed it in the UI in the settings to America/Chicago. 
I have updated settings.dist.py to set DD_TIME_ZONE=(str, 'America/Chicago')
I have added TZ: "America/Chicago" under the environment: section of every container in docker-compose.yml
I have verified that the timezone in the containers has changed to CST by running docker exec -it CONTAINERID date   and noting the timezone.
I have run dc-down.sh and dc-up.sh after making the changes to make sure i had rebuilt the containers. 

What else am I missing? When I go into a finding and update it and look at the history, it is always in UTC. I saw someone else had a similar issue a few months back but they seemed to have resolved it.

**Comment by acindm** [comment]
Maybe you should add the following steps:
1. Make sure to run `docker cp  /usr/share/zoneinfo/Asia/Shanghai <YOUR_CONTAINER_ID>:/etc/localtime `in every container.
2. Before running `dc-down.sh` and `dc-up.sh`, run `./dc-build.sh --no-cache`

Hope this is helpful to you and to those who find this page. :)
